src/api_routes_v4.rs

```diff
@@ -341,7 +341,12 @@ pub fn config(cfg: &mut ServiceConfig, rate_limit: &RateLimitCell) {
               .route("/list", get().to(list_media)),
           )
           .route("/inbox", get().to(list_inbox))
-          .route("/delete", post().to(delete_account))
+          // Use register rate limit for delete account
+          .service(
+            resource("/delete")
+              .wrap(rate_limit.register())
+              .route(post().to(delete_account)),
+          )
```

Comment:
even if the current plan is to remove api v3 before 1.0, given that #5516 isn't merged yet i suggest still adding the rate limit there as well in case it ends up not being removed.

Reply:
That PR should really be merged before this one anyway.